release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -175,6 +175,15 @@ For more information, see xref:../updating/understanding-upgrade-channels-releas
 [id="ocp-4-10-web-console"]
 === Web console
 
+[id="ocp-4-10-web-console-developer-perspective"]
+==== Developer perspective
+* You can now specify a name to a Service Binding connector in the *Topology* view while making a binding connection.
```

Comment:
```suggestion
* With this update, you can specify the name of a service binding connector in the *Topology* view while making a binding connection.
```
Lower case "service binding" unless you're talking about the "Service Binding Operator" proper noun.

Reply:
Thanks for pointing out the lower case. Will add this to the updates.